### PR TITLE
 [12] Style the Contact Us Page using CSS

### PR DESCRIPTION
**Pull request by Samju200**
## Task: `add style to contact page
contact-mobile 1
contact-mobile 2
contact-mobile 3
contact-tablet 1
contact-tablet 2
contact-tablet 3
contact-web 1
contact-web 2

`
- `added image to image folder and ed`

### CODE REVIEW

**Comment by adaugochi** [comment]
Also, you can change your font family to look way nicer. Visit google font and select your preferred font

**Comment by Samju200** [comment]
> Hello @Samju200
> Well done. Your work is amazing
> 
> But, I want you to create a separate styling file for the contact page. You can call it `contact.css`

Okay
Done
Thank you

**Comment by Samju200** [comment]
> Also, you can change your font family to look way nicer. Visit google font and select your preferred font

Okay
Done

File: submissions/adeyemiJuwon/css/contact.css [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+    margin: 19px 0px 32px 400px;
+}
+
+.contact-form .contact-send{
+    width: 113px;
+    background-color:rgb(255, 166, 95) ;
+}
+
+
+
```
Comment (adaugochi) [inline submissions/adeyemiJuwon/css/contact.css:134]: Remove too many white spaces. 2 lines are ok

File: submissions/adeyemiJuwon/css/contact.css [hunk 2]
```diff
@@ -0,0 +1,341 @@
...
+    flex-flow: column;
+}
+.contact-us{
+    margin-top: 20px;
+}
+.contact-form fieldset{
+    margin: 20px 30px 17px 30px ;
+    padding: 0 30px;
+    border-radius: 10px;
+
```
Comment (adaugochi) [inline submissions/adeyemiJuwon/css/contact.css:230]: remove spaces

File: submissions/adeyemiJuwon/css/contact.css [hunk 3]
```diff
@@ -0,0 +1,341 @@
...
+    
+}
+
+}
+
+
```
Comment (adaugochi) [inline submissions/adeyemiJuwon/css/contact.css:236]: remove spaces

File: submissions/adeyemiJuwon/css/contact.css [hunk 4]
```diff
@@ -0,0 +1,341 @@
...
+.contact-form .contact-send{
+    width: 50px;
+}
+textarea{
+    width: 350px;
+    height: 100px;
+}
+
+
+
```
Comment (adaugochi) [inline submissions/adeyemiJuwon/css/contact.css:326]: line 327 to the end isn't properly formated

Comment (Samju200) [reply submissions/adeyemiJuwon/css/contact.css:315]: what do you mean by not properly formated
Thanks

**Pull request merged**